stapi/display/display.tcl

```diff
@@ -229,14 +229,19 @@ catch { ::itcl::delete class STDisplay }
 		} else {
 			# TODO: this is not very good; should also hex-encode many other things.
 			foreach \
-				src " &		   {\"}		 <       > " \
+				src " &	       {\"}      <       > " \
```

Comment:
Yes, but I'm not going to do that here because:

1. It's existing code I'm not changing.
2. It's not used if you're running under rivet and this code always runs under rivet.